pkg/kvstore/etcd.go

```diff
@@ -469,27 +533,18 @@ func connectEtcdClient(config *client.Config, cfgPath string, errChan chan error
 	go func() {
 		defer close(errChan)
 
-		var session concurrency.Session
 		select {
 		case err = <-errorChan:
-			if err == nil {
-				session = *<-sessionChan
+			if err != nil {
+				errChan <- err
+				return
 			}
 		case <-time.After(initialConnectionTimeout):
-			err = fmt.Errorf("timed out while waiting for etcd session. Ensure that etcd is running on %s", config.Endpoints)
-		}
-		if err != nil {
-			errChan <- err
+			errChan <- fmt.Errorf("timed out while waiting for etcd session. Ensure that etcd is running on %s", config.Endpoints)
 			return
 		}
 
 		ec.getLogger().Debugf("Session received")
```

Comment:
Is this log message necessarily true anymore? We no longer receive a session.

Reply:
We still do when errorChan is closed on line 515